File: BasicCourseOOP/1. CursoPOOUber/Python/car.py
from driver import Driver
from user import User
from account import Account
class Car:
    id=int
    license=str
    # account=Account("","")/
    driver=Driver("","","","")
    user=User("","","","")
    passengers=int

    # def __init__(self,license,account):
    #     self.license=license
    #     self.account=account

    def __init__(self,license,driver):
        self.license=license
        self.driver=driver

    def __init__(self,license,user):
        self.license=license
        self.user=user

File: BasicCourseOOP/1. CursoPOOUber/Java/Car.java
class Car{
    Integer id;
    String license;
    Account account;
    Driver driver;
    User user;
    Integer passengers;

    public Car(String license, Account account){
        this.license=license;
        this.account=account;
    }

    public Car(String license, Driver driver){
        this.license=license;
        this.driver=driver;
    }

    public Car(String license, User user){
        this.license=license;
        this.user=user;
    }

    void printDataCar(){
        System.out.println("License: "+license+" Driver: "+account.name);
    }

    void printDataCarDriver(){
        System.out.println("License: "+license+" "+driver.printDataDriver());
    }

    void printDataUser(){
        System.out.println("License: "+license+" "+user.printDataUser());
    }
}